packages/api/amplify_api_android/android/src/main/kotlin/com/amazonaws/amplify/amplify_api/AmplifyApi.kt

```diff
@@ -54,105 +39,19 @@ class AmplifyApi : FlutterPlugin, MethodCallHandler, NativeApiPluginBindings.Nat
     }
 
     private var channel: MethodChannel? = null
-    private var eventchannel: EventChannel? = null
-    private var graphqlSubscriptionStreamHandler: GraphQLSubscriptionStreamHandler? = null
     private val logger = Amplify.Logging.forNamespace("amplify:flutter:api")
-    private var dispatcher: CoroutineDispatcher
-
-    constructor() {
-        dispatcher = Dispatchers.IO
-    }
-
-    @VisibleForTesting
-    constructor(
-        eventHandler: GraphQLSubscriptionStreamHandler,
-        dispatcher: CoroutineDispatcher = Dispatchers.IO
-    ) {
-        graphqlSubscriptionStreamHandler = eventHandler
-        this.dispatcher = dispatcher
-    }
-
-    private val handler = Handler(Looper.getMainLooper())
 
     override fun onAttachedToEngine(@NonNull flutterPluginBinding: FlutterPlugin.FlutterPluginBinding) {
-        graphqlSubscriptionStreamHandler = graphqlSubscriptionStreamHandler ?: GraphQLSubscriptionStreamHandler()
         channel = MethodChannel(flutterPluginBinding.binaryMessenger, "com.amazonaws.amplify/api")
```

Comment:
same question here as ios: Is this part of channel for auth token in oidc/lambda?